src/eredis.erl

```diff
@@ -123,9 +174,9 @@ start_link(Args) ->
 stop(Client) ->
     eredis_client:stop(Client).
 
--spec q(Client::client(), Command::[any()]) ->
-               {ok, return_value()} | {error, Reason::binary() | no_connection}.
-%% @doc: Executes the given command in the specified connection. The
+-spec q(Client::client(), Command::command()) ->
+               {ok, return_value()} | {error, Reason::term() | no_connection}.
```

Comment:
`Reason` is now as-generic as the underlying code is.

Reply:
+1